percona-packaging/scripts/psmdb_builder.sh

```diff
@@ -689,35 +694,32 @@ build_tarball(){
     #scons --prefix=$PWD/$PSMDIR install
     #
     mkdir -p ${PSMDIR}/bin
-    if [ ${DEBUG} = 0 ]; then
     for target in ${PSM_TARGETS[@]}; do
         cp -f $target ${PSMDIR}/bin
-        strip --strip-debug ${PSMDIR}/bin/${target}
+        if [ ${DEBUG} = 0 ]; then
+            strip --strip-debug ${PSMDIR}/bin/${target}
+	fi
     done
-    fi
     #
     cd ${WORKDIR}
     #
     # Build mongo tools
     cd ${TOOLSDIR}
+    mkdir -p build_tools/src/github.com/mongodb/mongo-tools
     rm -rf vendor/pkg
-    [[ ${PATH} == *"/usr/local/go/bin"* && -x /usr/local/go/bin/go ]] || export PATH=/usr/local/go/bin:${PATH}
-    export GOROOT="/usr/local/go/"
-    export GOPATH=$(pwd)/
-    export PATH="/usr/local/go/bin:$PATH:$GOPATH"
-    export GOBINPATH="/usr/local/go/bin"
-    . ./set_gopath.sh
+    cp -r $(ls | grep -v build_tools) build_tools/src/github.com/mongodb/mongo-tools/
+    cd build_tools/src/github.com/mongodb/mongo-tools
     . ./set_tools_revision.sh
-    mkdir -p bin
-    for i in bsondump mongostat mongofiles mongoexport mongoimport mongorestore mongodump mongotop mongooplog mongoreplay; do
-    echo "Building ${i}..."
+    sed -i 's|VersionStr="$(git describe)"|VersionStr="$PSMDB_TOOLS_REVISION"|' set_goenv.sh
+    sed -i 's|Gitspec="$(git rev-parse HEAD)"|Gitspec="$PSMDB_TOOLS_COMMIT_HASH"|' set_goenv.sh
+    . ./set_goenv.sh
```

Comment:
`. ./set_goenv.sh` is also already included in build.sh
I will leave it up to you if you want to remove it or not.